examples/plotting/file/linked_brushing.py

```diff
@@ -1,3 +1,11 @@
+'''Two scatter plots one representing # Cylinders vs. MPG and the second graph representing acceleration vs. MPG
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.Figure.circle
```

Comment:
Since this example uses the `autompg` sample data set, this should be included in the metadata as well please.